### PR TITLE
Specify unit string when sending gluster-block command when operating on gluster-block mode.

### PR DESCRIPTION
**Pull request by humblec**
Issue# https://github.com/kubernetes-incubator/external-storage/issues/404
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go [hunk 1]
```diff
@@ -361,7 +361,7 @@ func (p *glusterBlockProvisioner) createVolume(volSizeInt int, blockVol string,
 		// Execute gluster-block command.
 		cmd := exec.Command(
 			config.opMode, "create", config.blockModeArgs["glustervol"]+"/"+blockVol,
-			"ha", haCountStr, config.blockModeArgs["hosts"], sizeStr, "--json")
+			"ha", haCountStr, config.blockModeArgs["hosts"], sizeStr+"GB", "--json")
```
Comment (rootfs) [inline gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go]: do you need to check if sizeStr already has `G` or `GB` in it?

Comment (humblec) [reply gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go]: Actually its not required as the sizestr is derived by `roundupsize` which use the value of `resource request` in bytes. Thanks

**Comment by humblec** [comment]
@wongma7 can you please merge this PR ?

**Pull request merged**